4_arm.ipynb

```diff
@@ -194,13 +348,37 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "In addition, in \"Motion generation using forward kinematics\", only the HSR arm was used. However,  we cannot grab distant objects by moving the arm alone.\n",
+    "\n",
+    "HSR is a \"mobile manipulator\" that has the arm mounted on the move base. We will show the combined control using the move base and the arm from now on."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "ロボットの手先(エンドエフェクタ)の名前を確認します。アームの先にあるハンドのリンクが設定されているはずです+。\n",
     "\n",
     "ここで`whole_body`はロボットの頭を除く全関節角を管理する変数です。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Check the name of the robot's minions (=end effectors). The hand link at the end of the arm should be set.\n",
+    "\n",
+    "Here, `whole_body` is a variable that handles all joint angles of the robot except head."
```

Comment:
"Here, `whole_body` is a variable that handles all joint angles of the robot except for the head."